network-api/networkapi/donate_banner/models.py

```diff
@@ -64,15 +95,19 @@ class DonateBanner(TranslatableMixin, PreviewableMixin, models.Model):
         FieldPanel("subtitle"),
         FieldPanel("cta_button_text"),
         FieldPanel("cta_link"),
-        FieldPanel("background_image"),
+        FieldPanel("foreground_image"),
+        FieldPanel("bg_image"),
+        FieldPanel("bg_color"),
+        FieldPanel("text_color"),
     ]
 
     translatable_fields = [
         TranslatableField("title"),
         TranslatableField("subtitle"),
         TranslatableField("cta_button_text"),
         SynchronizedField("cta_link"),
-        SynchronizedField("background_image"),
+        SynchronizedField("foreground_image"),
+        SynchronizedField("bg_image"),
```

Comment:
```suggestion
        SynchronizedField("background_image"),
```

Reply:
I opted for using the `bg_`prefix instead of `background_` since when running the migration that renamed and added again the `background_image` field, database issued an error about the relationship already existing, since that field existed previously. I will change that and let you know if it still complains or if it was something related to db state in that moment.